core/pom.xml

```diff
@@ -22,7 +22,7 @@
     <parent>
         <groupId>org.moditect</groupId>
         <artifactId>moditect-parent</artifactId>
-        <version>1.0.0-SNAPSHOT</version>
+        <version>1.0.0-negatron</version>
```

Comment:
This shouldn't change (same in a few other POMs).

Reply:
I thought that you abandonned your project and I was preparing to go solo with my own fork :) It seems like I'm wrong so I'll rollback this particular change.

Reply:
done